prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -51,12 +51,12 @@ class SendFunction(private val workflowEngine: WorkflowEngine = WorkflowEngine()
                             is SFTPTransportType -> {
                                 workflowEngine
                                     .sftpTransport
-                                    .send(service.fullName, transport, content, reportId, retryItems)
+                                    .send(service, transport, content, reportId, retryItems, context)
                             }
```

Comment:
Now passing the entire OrganizationService object `service` instead of just the name.  Also passing the Azure Function context, so we can do logging.